src/TraceEvent/TraceEvent.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
It would be nice to use the TraceEvent library down to .NET Core 1.0. The ability to harness, measure performance, and compare between 1.0 and 2.0 runtimes is important for the dotnet family (e.g. CoreClr, CoreFx), where we track the product performance over time on the different released branches and master.